.github/workflows/tests.yml

```diff
@@ -311,11 +311,24 @@ jobs:
           restore-keys: ${{ runner.os }}-parallel_runtime_rspec-
 
       - name: Run brew tests
-        run: brew tests --online --coverage
+        run: |
+          # Retry multiple times when using BuildPulse to detect and submit
+          # flakiness (because rspec-retry is disabled).
+          if [ -n "$HOMEBREW_BUILDPULSE_ACCESS_KEY_ID" ]; then
+            brew tests --online --coverage || \
```

Comment:
A quick question to check my understanding of things: Am I right in thinking that each call to `brew tests` will run the [`Homebrew.tests` method](https://github.com/Homebrew/brew/blob/19e059235c1298ec5351f41af4a0479d4603e5a5/Library/Homebrew/dev-cmd/tests.rb#L65-L198)?

Reply:
@jasonrudolph Yup!